src/config.rs

```diff
@@ -0,0 +1,25 @@
+use clap::Parser;
+
+#[derive(Parser, Clone, Debug)]
+pub struct RunOptions {
+    /// The API key to use for auction server authentication.
+    #[arg(long = "pythnet-url", env = "PYTHNET_URL")]
+    pub pythnet_url:         String,
+    /// Path to the file containing the secret key.
+    #[arg(long = "secret-key", env = "SECRET_KEY")]
+    pub secret_key_path:     String,
+    /// The Wormhole program ID.
+    #[arg(
+        long = "wormhole-pid",
+        env = "WORMHOLE_PID",
+        default_value = "H3fxXJ86ADW2PNuDDmZJg6mzTtPxkYCpNuQUTgmJ7AjU"
+    )]
+    pub wormhole_pid:        String,
+    /// The address of the accumulator contract.
+    #[arg(
+        long = "accumulator-address",
+        env = "ACCUMULATOR_ADDRESS",
+        default_value = "G9LV2mp9ua1znRAfYwZz5cPiJMAbo1T6mbjdQsDZuMJg"
+    )]
+    pub accumulator_address: String,
```

Comment:
Now seeing this again I have a feeling that guardians might be happier if it's not there.

Reply:
actually this is the same in both Pythtest and Pythnet. So maybe we can just remove it and use the default :?